torch/hub.py

```diff
@@ -382,7 +382,19 @@ def load(github, model, *args, **kwargs):
 
 def _download_url_to_file(url, dst, hash_prefix, progress):
     file_size = None
-    u = urlopen(url)
+    # TODO: This is to get around CA issues on Python2, where urllib can't
```

Comment:
isn't this screwing over everyone in Python2 on security?
Remember `torch.hub` code actually has `eval`

Reply:
Can we add the `certifi` dependency?

Reply:
It seems like `requests` can successfully verify the cert in my experimentation on the docker container. Maybe we should just switch to that?

Reply:
yea maybe we add `requests` as a dependency for Python2 if that's the safest thing to do.

Reply:
like here: https://github.com/pytorch/pytorch/blob/master/setup.py#L353

Reply:
@soumith I added the dependency there, but how do I actually get the package installed in the CI test jobs?